content/en/llm_observability/setup/auto_instrumentation.md

```diff
@@ -137,12 +143,178 @@ The Google Gemini integration instruments the following methods:
 [9]: https://docs.anthropic.com/en/api/client-sdks#python
 [10]: https://docs.anthropic.com/en/api/messages
 [11]: https://docs.anthropic.com/en/api/messages-streaming
-[12]: /llm_observability/setup/sdk/#in-code-setup
+[12]: /llm_observability/setup/sdk/python/#in-code-setup
 [13]: https://python.langchain.com/v0.2/docs/concepts/#llms
 [14]: https://python.langchain.com/v0.2/docs/concepts/#chat-models
 [15]: https://python.langchain.com/v0.2/docs/concepts/#runnable-interface
-[16]: /llm_observability/setup/sdk/
+[16]: /llm_observability/setup/sdk/python
 [17]: https://python.langchain.com/v0.2/docs/concepts/#embedding-models
 [18]: /llm_observability/setup/sdk/#tracing-spans
 [19]: https://ai.google.dev/gemini-api/docs
 [20]: https://ai.google.dev/api/generate-content#method:-models.streamgeneratecontent
+{{% /tab %}}
+{{% tab "Node.js" %}}
+
+## Overview
+
+Datadog's [LLM Observability Node.js SDK][4] provides integrations that automatically trace and annotate calls to LLM frameworks and libraries. Without changing your code, you can get out-of-the-box traces and observability for calls that your LLM application makes to the following frameworks:
+
+
+| Framework                               | Supported Versions | Tracer Version       |
+|-----------------------------------------|--------------------|----------------------|
+| [OpenAI](#openai) (common JS)           | >= 3.0.0           | >= 4.49.0, >= 5.25.0 |
+
+In addition to capturing latency and errors, the integrations capture the input parameters, input and output messages, and token usage (when available) of each traced call.
+
+## Enabling and disabling integrations
+
+All integrations are enabled by default.
+
+To disable all integrations, use the [in-code SDK setup][6] and specify `plugins: false` on the general tracer configuration.
+
+```javascript
+const tracer = require('dd-trace').init({
+  llmobs: { ... },
+  plugins: false
+});
+const { llmobs } = tracer;
+```
+
+To only enable specific integrations:
+1. Use the [in-code SDK setup][6], specifying `plugins: false`.
+2. Manually enable the integration with `tracer.use()` at the top of the entrypoint file of your LLM application:
+
+```javascript
+const tracer = require('dd-trace').init({
+  llmobs: { ... },
+  plugins: false
+});
+
+const { llmobs } = tracer;
+tracer.use('<INTEGRATION_NAME_IN_LOWERCASE>', true);
```

Comment:
It looks like Python uses `<INTEGRATION_NAME_IN_LOWERCASE>` but I love putting real values in examples that actually work and require less grokking of the example. Also I don't like the implication that the user has to go find out the right magic string to make this work.

```suggestion
tracer.use('openai', true);
tracer.use('langchain', true);
// etc
```

Reply:
i like that too! will change for both node.js and python